tests/playbooks/tests_bond.yml

```diff
@@ -93,5 +97,9 @@
             - command: ip link del {{ controller_device }}
               ignore_errors: true
             - import_tasks: tasks/remove_test_interfaces_with_dhcp.yml
+            - name: "Restore the /etc/resolv.conf for initscript"
+              command: mv -vf /etc/resolv.conf.bak /etc/resolv.conf
+              when:
+                - network_provider == "initscripts"
```

Comment:
I think this task needs a `changed_when: false` but I'm not sure why it wasn't flagged by ansible-lint

Reply:
I guess it doesn't matter since this is in test code, and so will always report `changed` for the task, and will never be run again, so it won't run afoul of idempotency.